events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+            charToEntity,
+            entityToChar;
+    
+        function resetCharacterEntities() {
+            charToEntity = {};
+            entityToChar = {};
+            // add the default set
+            addCharacterEntities({
+                '&amp;': '&',
+                '&gt;': '>',
+                '&lt;': '<',
+                '&quot;': '"',
+                '&#39;': "'",
+                '&lsquo;': '"',
+                '&rsquo;': '"',
+                '&ldquo;': '"',
+                '&rdquo;': '"',
+                '&eacute;': 'e',
+                '&prime;': '\'',
+                '&Prime': '"'
+            });
+        }
+    
+        function addCharacterEntities(newEntities) {
+            var charKeys = [],
+                entityKeys = [],
+                key, encodedChar;
+            for (key in newEntities) {
+                encodedChar = newEntities[key];
+                entityToChar[key] = encodedChar;
+                charToEntity[encodedChar] = key;
+                charKeys.push(encodedChar);
+                entityKeys.push(key);
+            }
+            charToEntityRegex = new RegExp('(' + charKeys.join('|') + ')', 'g');
+            entityToCharRegex = new RegExp('(' + entityKeys.join('|') + '|&#[0-9]{1,5};' + ')', 'g');
+        }
+    
+        function htmlEncode(value) {
+            var htmlEncodeReplace = function(match, capture) {
+                return charToEntity[capture];
+            };
+    
+            return (!value) ? value : String(value).replace(charToEntityRegex, htmlEncodeReplace);
+        }
+    
+        function htmlDecode(value) {
+            var htmlDecodeReplaceFn = function(match, capture) {
+                return (capture in entityToChar) ? entityToChar[capture] : String.fromCharCode(parseInt(capture.substr(2), 10));
+            };
+    
+            return (!value) ? value : String(value).replace(entityToCharRegex, htmlDecodeReplaceFn);
+        }
+    
+        resetCharacterEntities();
+    
+        return {
+            htmlEncode: htmlEncode,
+            htmlDecode: htmlDecode
+        };
+    })();
+
+    function begin() {
+        findTargets();
+        bubbleOn();
+        prizeCalculator("new game");
+        if (!introPlayed) {
+            Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['GAME_INTRO']);
+            introPlayed = true;
+        } else {
+            Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['NEW_GAME_SFX']);
+        }
+        Entities.callEntityServerMethod(gameZoneProperties.id, "stopConfetti");
+        lights.forEach(function(light) {
+            Entities.callEntityServerMethod(light, "lightsOn");
+        }); 
+    }
+
+    function bubbleOn() {
+        Entities.callEntityServerMethod(bubble, "bubbleOn");
+        Messages.sendMessage(TRIVIA_CHANNEL, JSON.stringify({ 
+            type: "game on" 
+        }));
+        intervalBoard = Script.setInterval(function(){
```

Comment:
via code inspection, it seems possible that this interval might be set up more than once. you might want to check if this is already defined, and clear it if so

Reply:
fixed